src/pkg/utils/utils.go

```diff
@@ -40,7 +40,7 @@ import (
 
 const (
 	idTruncLength          = 12
-	releaseDefaultFallback = "30"
+	releaseDefaultFallback = "32"
```

Comment:
Ever since the default release for non-Fedora hosts was added in commit ca9ca9a3ded056408eb4934a6905b2488152b166, we have always pointed it to the oldest supported Fedora release.

I don't think there's any real reason for this. It's just what @andrunko went with, and after that @juhp also stuck with the same convention. So, unless there's a strong reason to change it, let's stick with the pattern.

Reply:
I took the liberty to tweak this, and merged.

Reply:
Very much fine with this, I simply didn't know about that policy.  Thank you for the fast work on this!